app/services/create_sponsorship.rb

```diff
@@ -0,0 +1,34 @@
+class CreateSponsorship
+
+  attr_reader :sponsorship, :sponsor, :orphan
```

Comment:
you can use `@sponsorship.save!` instead of `sponsorship.save!`. Not sure which is more preferred.